### ISSUE TITLE
Status of this document really final?

### ISSUE THREAD

**Issue by Tobias4872** [InteractiveAdvertisingBureau/OpenDirect#6]
Hi,

I am trying to wrap my mind around the status of this document. It says final, but carefully reading through it, it is easy to discover typos, broken links, outdated table of contents and other short comings. I wonder if there is going to be a "final final" version of this?

Thanks,
T.

**Comment by codeandcats** [comment]
Agreed. I just discovered some dead links in the table of contents and came to raise an issue (no specification for an `Ad` object)

**Comment by yuriy-yarosh** [comment]
Well, some of it you can guess yourself... while everything else is just try and error.
It's an informal spec without any valid audit mechanisms and integration patterns.

At least a start towards common API standard, still IAB does pretty mediocre job with their specs.

```
Product entity
retirementaate => retirementdate
deliverytype => is an object from 1.5.1 spec
```
